src/frontend/ExtensionManager/index.tsx

```diff
@@ -26,27 +27,48 @@ const ExtensionManager = function () {
   const rootRef = useRef<HTMLDivElement>(null)
   const trueAsStr = 'false' as unknown as boolean | undefined
 
+  const mmContainerStyle = {} as React.CSSProperties
+
+  if (OverlayState.title === 'HyperPlay Extension') {
+    mmContainerStyle.left = 0
+    mmContainerStyle.right = 'unset'
+    mmContainerStyle.top = 0
+    // document.documentElement.style.setProperty('--top-navbar-height', `0px`)
+  } else if (
+    OverlayState.title === 'HyperPlay Extension Overlay' ||
+    OverlayState.title === 'HyperPlay Browser Game' ||
+    OverlayState.title === 'HyperPlay Web Game'
```

Comment:
setting the window title with electron is flaky/buggy, so we can pass a title string to the frontend mobx class to indicate what type of overlay this is